packages/platform/types/event-types/event-types_2024_06_14/inputs/create-event-type.input.ts

```diff
@@ -102,6 +103,11 @@ export class CreateEventTypeInput_2024_06_14 {
   @IsInt()
   @Min(1)
   offsetStart?: number;
+
+  @IsOptional()
+  @ValidateNested()
+  @Type(() => RecurringEvent_2024_06_14)
+  recurringEvent?: RecurringEvent_2024_06_14;
```

Comment:
What do you think about renaming `recurringEvent` to `recurrence` and `RecurringEvent_2024_06_14` to `Recurrence_2024_06_14` ?

I am thinking about it because the input class is already for an event and recurrance is a characteristic of an event, but recurringEvent to me sounds more like a higher level description like it could be used for class name for example instead of being used as a property name.

What do you think?

Reply:
Sounds Good, will make changes

Reply:
@supalarry made the changes